docs/quickstart.rst

```diff
@@ -273,23 +282,21 @@ Download Sample Data
-
-  for url in ${URL_LIST}; do
-    curl -O "${url}"
-  done
-
-.. note::
-
-  The dataset is pending migration from PO.DAAC to the Earthdata Cloud in AWS. Following this migration, the above links may not work (though an updated download method will soon follow)
+  curl -O https://raw.githubusercontent.com/RKuttruff/incubator-sdap-nexus/SDAP-451-qs-update/docs/granule-download.sh
```

Comment:
Thanks for making this change @RKuttruff. Can you please reference the master incubator-sdap-nexus branch instead? I know the file doesn't exist yet but it will after we merge.

Reply:
Updated.